content/handbook/Interim_processes/decision_making_process.md

```diff
@@ -0,0 +1,56 @@
+
+## Consensus Agreement for Initial Major Decisions
+Initially, our major decisions will be about how we function as a collective, including the principles informing our collective practices and how we make different types of decisions together. 
+
+While there are many ways of forming consensus agreements, to start us off we are adopting a 4-stage version: 
+
+1. INTRODUCTION, OPEN DISCUSSION, & EXPLORATION:   
+A facilitator starts a thread on Loomio, introducing an upcoming decision-point for the collective and open a question round discussion to clarify any points, express different perspectives, and explore potential pathways forward. If an Assembly occurs during this time, the topic will be included on the agenda for some in-person sense-making.
+2. PROPOSAL & AMENDMENTS:   
+Following discussion, the facilitator will make a proposal and run a sense check (e.g., a [gradients-of-agreement poll](https://help.loomio.com/en/user_manual/polls/proposals/index.html#gradients-of-agreement)) to encourage discussion of any concerns. If an Assembly occurs during this time, the topic will be included on the agenda for some in-person sense-making. The facilitator can amend the proposal as needed and run another sense-check.
+3. AGREEMENT-TEST   
+When the proposal is stable, the facilitator will start a formal ['consensus proposal'](https://help.loomio.com/en/user_manual/polls/proposals/index.html#consensus-proposal) on Loomio to test for agreement. Note that those who choose to [block](https://www.seedsforchange.org.uk/consensus#block) agreement at this stage will be expected to take responsibility for updating the proposal and facilitating a new round of decision-making. 
+4. IMPLEMENTATION:  
+The facilitator summarises the outcome and initiate any steps required for implementation (e.g., updating the handbook)
+
+
+## Consent Agreement for Initial Minor Decisions
+Minor decisions are those with low-risk and easily-reversible outcomes. Examples of minor decisions include selecting professionals who can review our T2S model, choosing a location for hosting our next assembly, prioritising the next tasks needed to move the project forward, selecting a process for documenting our decisions, etc.,   
+
+Our interim process for making minor decisions is to delegate decisions types to the crew through a *consent agreement* process. In this context, a *consent agreement* is similar to consensus, but instead of seeking the best decision for the group, consent is the absence of objections ('I can live with this decisions'). Rather then aiming to resolve all concerns, the proposal is only amended for 'principled objections' (such as “I think there is a serious risk this proposal could do harm”, not “I don’t like it”). 
```

Comment:
I prefer: "to a crew" rather than "to the crew"

Reply:
There doesn't seem need to put the word "agreement" on the end of consent here. It's interesting seeing it compared to consensus without it, as in "a *consent agreement* is similar to consensus". That could just be "consent is similar to consensus"